chromium_src/chrome/browser/about_flags.cc

```diff
@@ -22,6 +24,24 @@ using ntp_background_images::features::kBraveNTPBrandedWallpaper;
 using ntp_background_images::features::kBraveNTPBrandedWallpaperDemo;
 using ntp_background_images::features::kBraveNTPSuperReferralWallpaper;
 
+#define BRAVE_SKIP_CONDITIONAL_FEATURE_ENTRY                               \
```

Comment:
maybe just cut it away from compilation via gn? By checking `brave_channel` and setting `ENABLE_PLAYLIST` accordingly

Reply:
At first, that way was used. Only buildflag was on for development and nightly channel.
However, as @bridiver mentioned earilier(https://github.com/brave/brave-core/pull/5733#discussion_r458345460), we couldn't catch some build failure.
So, using channel info was removed from buildflag configuration. WDYT?

Reply:
yeah, fair enough. In general I think we should launch our own Finch instance :)

Reply:
can we override IsFlagExpired here instead of patching?

Reply:
@bridiver Yup, changed to use override.